packages/web-app-files/src/composables/sort/useSort.ts

```diff
@@ -29,9 +29,9 @@ export interface SortOptions<T extends SortableItem> {
 }
 
 export interface SortResult<T> {
-  items: ComputedRef<Array<T>>
-  sortBy: ComputedRef<string>
-  sortDir: ComputedRef<SortDir>
+  items: MaybeRef<Array<T>>
```

Comment:
Note: this is a MaybeRef instead of a ReadOnlyRef because we use a Partial of this type as argument to a Mock Factory and it makes passing along the value easier